25.md

```diff
@@ -7,20 +7,15 @@ Reactions
 
 `draft` `optional`
 
-A reaction is a `kind 7` event that is used to react to other events.
+A reaction is a `kind 7` event that is used to indicate user reactions to other events. A
+reaction's `content` field MUST include user-generated-content indicating the value of the
+reaction.
 
-The generic reaction, represented by the `content` set to a `+` string, SHOULD
-be interpreted as a "like" or "upvote".
+A reaction with `content` set to `+` or an empty string MUST be interpreted as a "like" or "upvote".
+A reaction with `content` set to `-` MUST be interpreted as a "dislike" or "downvote".
 
-A reaction with `content` set to `-` SHOULD be interpreted as a "dislike" or
-"downvote". It SHOULD NOT be counted as a "like", and MAY be displayed as a
-downvote or dislike on a post. A client MAY also choose to tally likes against
-dislikes in a reddit-like system of upvotes and downvotes, or display them as
-separate tallies.
-
-The `content` MAY be an emoji, or [NIP-30](30.md) custom emoji in this case it MAY be interpreted as a "like" or "dislike",
-or the client MAY display this emoji reaction on the post. If the `content` is an empty string then the client should
-consider it a "+".
+A reaction with `content` set to an emoji or [NIP-30](30.md) custom emoji SHOULD NOT be interpreted
```

Comment:
Now there is nothing that explicitly says content can be an emoji. It's only implied

Reply:
Added a small clarification.